pkg/apis/infinispan/v1/types_util.go

```diff
@@ -90,6 +90,9 @@ func (ispn *Infinispan) ApplyDefaults() {
 		cpuLimitString := comutil.ToMilliDecimalQuantity(constants.DefaultCPULimit)
 		ispn.Spec.Container.CPU = cpuLimitString.String()
 	}
+	if ispn.Spec.Service.Container.Storage == "" {
+		ispn.Spec.Service.Container.Storage = consts.DefaultPVSize.String()
```

Comment:
The storage size is make sense for DataGrid only in current impl. Do we need to set defaults for all cases.
Otherwise, we also validate that `m.Spec.Service.Container.Storage != ""` and this is doesn't make sense now.